--- CMakeLists.txt.orig	2013-02-19 16:32:57.215779326 +0000
+++ CMakeLists.txt	2013-02-19 16:34:19.346782650 +0000
@@ -537,7 +537,7 @@
 #    )
     install(FILES
       "${CMAKE_CURRENT_BINARY_DIR}/avogadro.prf"
-      DESTINATION "${CMAKE_INSTALL_PREFIX}/${QT_MKSPECS_RELATIVE}/features"
+      DESTINATION "${QT_MKSPECS_RELATIVE}/features"
     )
 #  endif ("${QT_MKSPECS_DIR}" MATCHES ^$"${CMAKE_INSTALL_PREFIX}/")
 else()
