--- CMakeLists.txt.qt4	2011-05-18 15:54:11.000000000 +0200
+++ CMakeLists.txt	2011-05-18 15:54:52.000000000 +0200
@@ -491,7 +491,7 @@
 if(QT_MKSPECS_DIR AND INSTALL_QMAKE_PRF)
     install(FILES
       "${CMAKE_CURRENT_BINARY_DIR}/avogadro.prf"
-      DESTINATION "${CMAKE_INSTALL_PREFIX}/${QT_MKSPECS_RELATIVE}/features"
+      DESTINATION "${QT_MKSPECS_DIR}/features"
     )
 else(QT_MKSPECS_DIR AND INSTALL_QMAKE_PRF)
   message(WARNING "Qt mkspecs directory not found. File 'avogadro.prf' will be installed to ${SHARE_INSTALL_PREFIX}/avogadro/qmake.")
