--- avogadro-1.1.1/libavogadro/src/extensions/qtaim/qtaimcubature.cpp.omv~	2014-01-22 00:58:04.186160909 +0100
+++ avogadro-1.1.1/libavogadro/src/extensions/qtaim/qtaimcubature.cpp	2014-01-22 00:58:24.556160265 +0100
@@ -1189,7 +1189,8 @@ QList<QVariant> QTAIMEvaluateProperty(QL
 
   QList<QVariant> valueList;
 
-  double initialElectronDensity=eval.electronDensity( Eigen::Vector3d(x0,y0,z0) );
+  // Using auto because it might be float or double, depending on architecture
+  auto initialElectronDensity=eval.electronDensity( Eigen::Vector3d(x0,y0,z0) );
 
   // if less than some small value, then return zero for all integrands.
   if( initialElectronDensity < 1.e-5 )
